lib/pdksync.rb

```diff
@@ -133,6 +132,23 @@ def self.clone_directory(namespace, module_name, output_path)
     puts "(FAILURE) Cloning for #{module_name} failed - check the module name and namespace are correct."
   end
 
+  # @summary
+  #   This method when called will run the 'pdk convert' command at the given location, with an error message being thrown if it is not successful.
+  # @param [String] output_path
+  #   The location that the command is to be run from.
+  # @return [Integer]
+  #   The status code of the pdk converty run.
+  def self.pdk_convert(output_path)
+    Dir.chdir(output_path) unless Dir.pwd == output_path
+    _stdout, stderr, status = Open3.capture3('pdk convert --force --template-url https://github.com/puppetlabs/pdk-templates')
+    if status != 0
+      puts "(FAILURE) Unable to run `pdk convert`: #{stderr}"
+    else
+      puts 'PDK Convert has run.'
```

Comment:
Possibly a more formative message here, stating that is has been converted against https://github.com/puppetlabs/pdk-templates or something?

Reply:
Also you may want to add the additional functionality in this function to cope with there being no changes required. Example can be found in pdk_update function.